superset-frontend/src/dashboard/components/PropertiesModal.jsx

```diff
@@ -346,6 +347,15 @@ class PropertiesModal extends React.PureComponent {
               >
                 {saveLabel}
               </Button>
+              <Button
```

Comment:
This looks like a rebase error or something, as there are now TWO cancel buttons here.

Reply:
yes I deleted this now